treeherder/embed/views.py

```diff
@@ -11,13 +11,14 @@ class PushStatusView(TemplateView):
 
     template_name = "embed/push_status.html"
 
-    def get_context_data(self, **kwargs):
-        assert "repository" in kwargs and "revision" in kwargs
-        (repository_name, revision) = (kwargs['repository'], kwargs['revision'])
-        context = super(PushStatusView, self).get_context_data(**kwargs)
+    def get_context_data(self, repository, revision):
```

Comment:
no test needed for this as already covered by django and would raise `NoReverseMatch` https://github.com/mozilla/treeherder/blob/master/treeherder/embed/urls.py#L10